test/sqllogictest/list.slt

```diff
@@ -1086,6 +1086,35 @@ SELECT list_length_max((LIST[1]::INT LIST), -1)
 query error invalid dimension: -1; must use value within \[1, 1\]
 SELECT list_length_max((LIST[1]::INT LIST), LIST[-1][1])
 
+# 🔬🔬 unnest
+
+query I
+SELECT * FROM unnest(LIST[1,2,3])
```

Comment:
So in `psql` I see that I can execute both:
```
jessicalaughlin=# select * from unnest(array[1,2]);
 unnest
--------
      1
      2
(2 rows)
```
and
```
jessicalaughlin=# select unnest(array[1,2]);
 unnest
--------
      1
      2
(2 rows)
```

Is this true of our table functions? If so, could you add a test for the second case?